File: PRAK602-2310817120012-RirinCitraLestari.py
def calculate_zetsu_rooms(n, zetsu):
    result = [zetsu[i] * (i + 1) for i in range(n)]
    print(*result)

def main():
    n = int(input())
    zetsu = list(map(int, input().split()))

    calculate_zetsu_rooms(n, zetsu)

if __name__ == "__main__":
    main()

File: PRAK602-2310817120012-RirinCitraLestari.c
#include <stdio.h>

void calculateZetsuRooms(int n, int zetsu[]) {
    for (int i = 0; i < n; i++) {
        printf("%d ", zetsu[i] * (i + 1));
    }
}

int main() {
    int n;
    scanf("%d", &n);

    int zetsu[n];

    for (int i = 0; i < n; i++) {
        scanf("%d", &zetsu[i]);
    }

    calculateZetsuRooms(n, zetsu);

    return 0;
}
